plugin/src/main/java/com/denizenscript/denizen/objects/ColorTag.java

```diff
@@ -475,10 +475,9 @@ public static void registerTags() {
         // @description
         // Returns the color that results if you mix this color with another.
         // -->
-        tagProcessor.registerTag(ColorTag.class, "mix", (attribute, object) -> { // Temporarily non-static because the input could be 'random'
-            ColorTag mixed_with = attribute.paramAsType(ColorTag.class);
-            if (mixed_with != null) {
-                return new ColorTag(object.mixWith(mixed_with));
+        tagProcessor.registerTag(ColorTag.class, ColorTag.class, "mix", (attribute, object, mixWith) -> { // Temporarily non-static because the input could be 'random'
+            if (mixWith != null) {

```

Comment:
Can remove `null` check and the error here as well

Reply:
Got it! 🙌 Pushing the change now